projects/storefrontapp-e2e-cypress/cypress/helpers/customer-ticketing/customer-ticketing-helpers/ticket-listing.ts

```diff
@@ -197,3 +198,44 @@ function getIdInRow(rowNumber: number) {
     .eq(ID_COLUMN)
     .find('a');
 }
+
+export function verifyTicketIdIsSmallerInNextPageComparedToPreviousPageByComparingIds(){
+  const TOTAL_NUMBER_OF_PAGES_TO_VISIT = 3;
+  for(let page = 1; page < TOTAL_NUMBER_OF_PAGES_TO_VISIT; page++){
+    getIdInRow(FIFTH_ROW_TICKET_LIST).then((id) => {
+      const smallerId = parseInt(id.text(), 10);
+      var next_page = page+1;
+      cy.get(`aria-label="page ${next_page}"`).click();
+      getIdInRow(FIFTH_ROW_TICKET_LIST)
+        .invoke('text')
+        .then(parseFloat)
+        .should('be.lt', smallerId);
+    });
+  };
+}
+
+export function verifyTicketIdIsSmallerInLastPageComparedToFirstPageByComparingIds(){
+  getIdInRow(FIRST_ROW_TICKET_LIST).then((id) => {
+    const smallerId = parseInt(id.text(), 10);
+    clickPageOnPagination("last");
+    getIdInRow(FIRST_ROW_TICKET_LIST)
+      .invoke('text')
+      .then(parseFloat)
+      .should('be.lt', smallerId);
+  });
+}
+
+export function clickPageOnPagination(pageName: string){
+  cy.get(`aria-label="${pageName} page"`).click();
```

Comment:
Label should be a better fit for "first, 1, 2, 3, ... "
```suggestion
export function clickPageOnPagination(pageLabel: string){
  cy.get(`aria-label="${pageLabel} page"`).click();
```

Reply:
Currently, the labels go as; first page, page 1, page 2,..., last page. Do you think it would be better to put at if-else statement or just leave the function the way it is?
```suggestion
export function clickPageOnPagination(pageLabel: string){
  if(pageLabel.length() > 1){
  cy.get(`aria-label="page ${pageLabel} "`).click();
  } else {
  cy.get(`aria-label="${pageLabel} page"`).click();
  }
```

Reply:
You can leave it as it is,  just update the variable name from pageName to pageLabel